MekHQ/data/scenariotemplates/Destroy Grounded Dropship.xml

```diff
@@ -169,8 +169,10 @@
                     <howMuch>2</howMuch>
                 </failureEffect>
             </failureEffects>
-            <additionalDetails/>
-            <description>Destroy, cripple or force to withdraw the following force(s) and unit(s):</description>
+            <additionalDetails>
+                <additionalDetail></additionalDetail>
+            </additionalDetails>
```

Comment:
Why the addition of this?

Reply:
I think it's a function of how the code is saved, there's probably a "don't serialize empty lists" flag somewhere.